ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml

```diff
@@ -0,0 +1,64 @@
+base_images:
+  base:
+    name: "4.3"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+images:
+- dockerfile_path: Dockerfile.rhel
+  from: base
+  inputs:
+    bin:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      paths: null
+  to: cluster-policy-controller
+# TODO: wait for image to exist
```

Comment:
promotion is fine even without ART (not related to CI) you just can't use the image in the code

Reply:
oh ok, i've uncommented